packages/swingset-liveslots/test/test-liveslots.js

```diff
@@ -427,7 +427,10 @@ test('liveslots vs symbols', async t => {
       },
     });
   }
-  const { dispatch } = await makeDispatch(syscall, build);
+  const { dispatch, testHooks } = await makeDispatch(syscall, build);
+  const { getRetentionStats } = testHooks;
+  console.log('@@@ pre:', getRetentionStats());
```

Comment:
let's remove these console.logs, or comment them out, our tests are noisy enough as it is :)

Reply:
Yeah, this one was actually a mistake on my part.  Eslint will complain if you try to checkin code containing a `debugger;` statement.  I always tag my console debug messages with `@@@` so that I can grep for them in a big blob of console spam; I wonder if I could tweak eslint to flag these...